tarpaulin.toml

```diff
@@ -1,4 +1,5 @@
 [default-config]
 debug = false
 verbose = false
-exclude-files = ["matrix_sdk/examples/*", "matrix_sdk_common", "matrix_sdk_test"]
+exclude-files = ["crates/matrix_sdk/examples/*", "crates/matrix_sdk_common", "crates/matrix_sdk_test"]
+exclude = ["matrix-sdk-crypto-js", "matrix-sdk-crypto-nodejs"]
```

Comment:
Why not add the `common` and `test` crate here as well if the `exclude` parameter is smart enough to handle crates?

We should also add the rest of the bindings crates and the `matrix-sdk-indexeddb` crate as well. Furthermore the `xtask` setup and the `labs` should be ignored as well.

![image](https://user-images.githubusercontent.com/552026/173319727-b9f25b74-72ab-44c6-bfa9-0cc9e16a9ef5.png)